packages/buttons/src/elements/Button.tsx

```diff
@@ -77,5 +77,21 @@ Button.defaultProps = {
   size: 'medium'
 };
 
+interface IIconProps {
+  isRotated?: boolean;
+  children: any;
+}
+
+const StartIcon = (props: IIconProps) => <StyledIcon position="start" {...props} />;
+const EndIcon = (props: IIconProps) => <StyledIcon position="end" {...props} />;
+
+(Button as any).StartIcon = StartIcon;
+(Button as any).EndIcon = EndIcon;
+
 /** @component */
-export default Button;
+export default Button as React.FunctionComponent<
+  IButtonProps & React.RefAttributes<HTMLButtonElement>
+> & {
+  StartIcon: typeof StartIcon;
+  EndIcon: typeof EndIcon;
```

Comment:
@austin94 I have a commit ready that renames these to `Start` and `End` if we decide that's the preferred API.

Reply:
This seems fine to me. Whichever feels best to you.